can_tools/scrapers/__init__.py

```diff
@@ -119,14 +119,7 @@
 from can_tools.scrapers.official.NC.nc_vaccine import NCVaccine
 from can_tools.scrapers.official.ND.nd_vaccines import NDVaccineCounty
 
-from can_tools.scrapers.official.NE.ne_vaccines import (
```

Comment:
the Nebraska portal now requires a login/authentication :( so our scrapers no longer work. 

The vaccine scrapers are state-level so I've just removed them. I'm leaving the NebraskaCases code here in case they begin to publish data publicly again at some point